Makefile

```diff
@@ -55,8 +55,7 @@ build-example: fmt bin build
 
 # formats the repo
 fmt:
-	@echo "Running 'go fmt ...'"
-	@go fmt -x "$(REPO)/..."
```

Comment:
This is the only place that depended on `go` outside of a container. Moving it into the container eliminates the dependency on `go` on CI nodes.